examples/chainfleetautoscaler.yaml

```
@@ -0,0 +1,100 @@
+---
+# Copyright 2024 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# [Stage:Dev]
+# [FeatureFlag:ScheduledAutoscaler]
+# Example of a FleetAutoscaler - this is used to scale a Fleet based on a schedule.
+#
+
+#
+# For a full reference and details: https://agones.dev/site/docs/reference/fleetautoscaler/
+#
+apiVersion: autoscaling.agones.dev/v1
+kind: FleetAutoscaler
+metadata:
+  name: chain-fleet-autoscaler
+spec:
+  policy:
+    # Chain based policy for autoscaling.
+    type: Chain
+    chain:
+      # UID of chain entry. Optional.
+      - uid: "weekday"
```

```suggestion
      - id: "weekday"
```

it's an `id` I don't think it needs a `u`.

Or `name` if we have prior art using `name` in a list item.